pages/DSNP/Identifiers.md

```diff
@@ -69,9 +70,9 @@ In order for DSNP applications to interoperate, the required functionality is li
 
 - CID version: MUST be version 1, in order to distinguish CIDs from simple multihash values in situations where either may be used
 - Hash algorithm: MUST be `sha2-256` or `blake2b-256`
-- Encoding: MUST be `base58btc` or `base32`
 - Codec: MUST be `dag-pb` for data 256*1024 bytes or longer; `raw` for data less than 256*1024 bytes
```

Comment:
Adding backslash before `*` helps markdown to disambiguate:

- Codec: MUST be `dag-pb` for data 256\*1024 bytes or longer; `raw` for data less than 256\*1024 bytes